docs/layers/base-tile-layer.md

```diff
@@ -81,6 +80,15 @@ The maximum cache size for a tile layer. If not defined, it is calculated using
 
 - Default: `null`
 
+##### `strategy` (Enum, optional)
+
+How the tile layer determines the visibility of tiles. One of the following:
+
+* `'default'`: If some tiles in the current viewport are waiting for data to load, aggresively use cached content from other zoom levels to fill the empty space.
+* `'exclusive'`: Avoid showing overlapping tiles when backfilling with cached content.
```

Comment:
- The two bullets are worded very differently (waiting for data to load/backfilling with cached content) but basically talk about the same thing, this makes it a little confusing.
- I'd prefer if both strategies were named, rather than just calling one `default`.
- in 3d tiles, the strategies are called `ADDITIVE` (all zoom levels drawn at once, used by point clouds) and `REPLACEMENT` (render parent tile until all child tiles are loaded, used by 3d models/photogrammetry).
- The 2d tile semantics are slightly different (i.e. we don't have to draw the entire parent chain in "additive" mode), but if they are close enough then we could call them the same thing, or at least name them in a similar way.
- Basically, I want one mode where we draw a parent tile and wait for all 4 children to load - before that we draw the children (and stop drawing the parent).

Since "default/ADDITIVE" doesn't work opacity is not 1, I propose that the mode would automatically be set to `REPLACEMENT` in those cases.

Reply:
I'm reluctant to use the name `additive` because it's not the same thing as what 3d tiles uses.

How do you feel about calling this prop `replacementStrategy`, with two possible values:

- `'coverage'`: prioritize showing the best available content for every pixel
- `'no-overlap'`: avoid showing overlapping tiles